File: samples/SignalR.Hosting.Owin.Samples/Hubs/MouseTracking/MouseTracking.cs
using System.Threading;
using SignalR.Hubs;

namespace SignalR.Samples.Hubs.MouseTracking
{
    public class MouseTracking : Hub
    {
        private static long _id;

        public void Join()
        {
            Caller.id = Interlocked.Increment(ref _id);
        }

        public void Move(int x, int y)
        {
            Clients.moveMouse(Caller.id, x, y);
        }
    }
}

File: samples/SignalR.Hosting.Owin.Samples/Content/Hubs/MouseTracking/MouseTracking.js
$(function () {
    // Pure client side hub
    var signalR = $.connection;
    signalR.mouseTracking.moveMouse = function (id, x, y) {
        if (id == this.id) {
            return;
        }

        updateCursor(id, x, y);
    };

    function updateCursor(id, x, y) {
        var e = document.getElementById(id);
        if (!e) {
            e = $('<div id="' + id + '" class="icon icon-wand">' + id + '</div>').appendTo(document.body);
            e.css('position', 'absolute');
        }
        else {
            e = $(e);
        }
        e.css('left', x);
        e.css('top', y);
    }


    signalR.hub.start({ transport: activeTransport }, function () {
        signalR.mouseTracking.join(function () {

            $(document).mousemove(function (e) {
                signalR.mouseTracking.move(e.pageX, e.pageY);
                updateCursor(signalR.mouseTracking.id, e.pageX, e.pageY);
            });
        });
    });
});